ssn/integrated/ssn.ttl

```diff
@@ -46,14 +46,66 @@ ssn: a owl:Ontology ;
   rdfs:isDefinedBy ssn: .
 
+  ssn:hasProperty a owl:ObjectProperty ; # TODO: need resolution, see https://www.w3.org/2015/spatial/wiki/Link_between_FeatureOfInterest_and_xxxProperty
+    rdfs:label "has property"@en ;
+    skos:definition """Relation between a FeatureOfInterest and a Property of that feature."""@en ;
+    owl:inverseOf ssn:isPropertyOf ;
+    rdfs:isDefinedBy ssn: .
+
+  ssn:isPropertyOf a owl:ObjectProperty ; # TODO: need resolution, see https://www.w3.org/2015/spatial/wiki/Link_between_FeatureOfInterest_and_xxxProperty
+    rdfs:label "is property of"@en ;
+    skos:definition """Relation between a FeatureOfInterest and a Property (a Quality observable by a sensor) of that feature."""@en ;
+    owl:inverseOf ssn:hasProperty ;
+    rdfs:isDefinedBy ssn: .
+
   sosa:ObservableProperty rdfs:subClassOf ssn:Property .
 
   sosa:ActuatableProperty rdfs:subClassOf ssn:Property .
 
 
+## Systems
+
+ssn:System a owl:Class ;
+  rdfs:label "System"@en ;
+  skos:definition """System is a unit of abstraction for pieces of infrastructure for sensing. A system has components, its subsystems, which are other systems."""@en ; # TODO: update with: "A system may have components" ? 
+  rdfs:subClassOf [ owl:onProperty sosa:isHostedBy ; owl:allValuesFrom sosa:Platform ] ;
+  rdfs:subClassOf [ owl:onProperty ssn:hasSubSystem ; owl:allValuesFrom ssn:System ] ;
+#  rdfs:subClassOf [ owl:onProperty ssn:hasDeployment ; owl:allValuesFrom ssn:Deployment ] ;
+#  rdfs:subClassOf [ owl:onProperty ssn:hasOperatingRange ; owl:allValuesFrom ssn:OperatingRange ] ;
+#  rdfs:subClassOf [ owl:onProperty ssn:hasSurvivalRange ; owl:allValuesFrom ssn:SurvivalRange ] ;
+  rdfs:isDefinedBy ssn: .
+
+  ssn:hasSubSystem a owl:ObjectProperty ;
+    rdfs:label "has subsystem"@en ;
+    skos:definition """Relation between a system and its component parts (has-part)."""@en ;
+    rdfs:isDefinedBy ssn: .
+
+
+## Platforms
+
+sosa:Platform 
+  rdfs:subClassOf [ owl:onProperty sosa:hosts ; owl:allValuesFrom ssn:System ] .
+#  rdfs:subClassOf [ owl:onProperty ssn:inDeployment ; owl:allValuesFrom ssn:Deployment ] ;
+
+
+## Sensor, Actuator
+
+sosa:Sensor rdfs:subClassOf ssn:System .
+#  rdfs:subClassOf [ owl:onProperty ssn:implements ; owl:someValuesFrom ssn:Sensing ] ;
+#  rdfs:subClassOf [ owl:onProperty ssn:detects ; owl:allValuesFrom ssn:Stimulus ] ;
+#  rdfs:subClassOf [ owl:onProperty ssn:hasMeasurementCapability ; owl:allValuesFrom ssn:MeasurementCapability ] ;
+#  rdfs:subClassOf [ owl:onProperty ssn:observes ; owl:allValuesFrom ssn:Property ] .
+
+sosa:Actuator rdfs:subClassOf ssn:System .
+#  rdfs:subClassOf [ owl:onProperty ssn:implements ; owl:someValuesFrom ssn:Acting ] ;
+#  rdfs:subClassOf [ owl:onProperty ssn:hasActuationCapability ; owl:allValuesFrom ssn:ActuationCapability ] ;
+#  rdfs:subClassOf [ owl:onProperty ssn:actsOn ; owl:allValuesFrom ssn:Property ] .
```

Comment:
use sosa:Actuator,
proposed some axioms to do some parallel with sosa;Sensor, 
these are commented for now, because they involve some terms that have not yet been integrated, and we need to discuss them individually.